plugins/WireframeView/wireframe.shader

```diff
@@ -0,0 +1,91 @@
+[shaders]
+vertex =
+    uniform highp mat4 u_model_view_projection_matrix;
+
+    attribute highp vec4 a_vertex;
+
+    void main()
+    {
+        gl_Position = u_model_view_projection_matrix * a_vertex;
+    }
+
+fragment =
+    uniform lowp vec4 u_color;
+
+    void main()
+    {
+        gl_FragColor = u_color;
```

Comment:
See OpenGLContext.detectBestOpenGLVersion()
I think the best way would be to use that to check for the right OpenGL version in the __init__.py register() method and return an empty dict when the gl version is too low.